code/modules/projectiles/projectile/bullets/revolver.dm

```diff
@@ -14,8 +14,9 @@
 
 /obj/item/projectile/bullet/c38
 	name = ".38 bullet"
-	damage = 25
+	damage = 15
 	//knockdown = 60 //yogs - commented out
+	stamina = 60

```

Comment:
Not modular

Reply:
what the fuck does this mean

Reply:
It means it needs to use the modular codebase format. You're editing pure /tg/ code which is kept intact and unedited for purposes of staying up to date. Yogs then has other code files that override the /tg/ code as needed. This is supposed to be one of those files.

Reply:
I guess you don't need to do that for one line changes, however. So ignore me, sorry.